columnflow/tasks/external.py

```diff
@@ -156,6 +156,7 @@ def iter_nano_files(
         fs: str | Sequence[str] | None = None,
         lfn_indices: list[int] | None = None,
         eager_lookup: bool | int = 1,
+        skip_fallback: bool = False,
```

Comment:
Can you add this parameter to the doc string?

Reply:
Added.